src/containers/EditEventPage.js

```diff
@@ -20,8 +21,14 @@ class EditEventPage extends React.Component {
   render() {
     return (
       <>
-        <EventForm event={this.props.event} onSubmit={this.onSubmit} />
-        <button onClick={this.onRemoveClick}>Remove event</button>
+        {this.props.event ? (
+          <>
+            <EventForm event={this.props.event} onSubmit={this.onSubmit} />
+            <button onClick={this.onRemoveClick}>Remove event</button>{" "}
```

Comment:
`{" "}` ? 🤔

Reply:
I don't know why but sometimes it is added automatically by vsc